docs/windows.md

```diff
@@ -154,7 +154,9 @@ kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/
 Antrea provides a pre-built OVS package which contains test-signed OVS kernel
 driver. If you don't have a self-signed OVS package and just want to try the
 Antrea on Windows, this package can be used for testing. We also provide a help
-script to install the OVS driver and register userspace binaries as services.
+script `Install-OVS.ps1` to install the OVS driver and register userspace binaries
+as services. You can set `-ImportCertificate` option of `Install-OVS.ps1` to $false
+if you have a signed OVS package.
```

Comment:
Indeed.
I didn't separate into 2 sections since they share many. But I still mentioned the difference (test-only and production) and I think it is enough for users to distinguish. PTAL.